CITATION.rst

```diff
@@ -10,4 +19,15 @@ specific version that you used. We also encourage you to cite the software DOI
 in the main text wherever appropriate. DOIs and BibTeX keys for each release
 can be found on Zenodo_.
 
+  .. only:: builder_html
+
+     .. container:: sphx-glr-footer
+        :class: sphx-glr-footer-gallery
+
+      .. container:: sphx-glr-download sphx-glr-download-python
+
+         :download:`Download BibTeX <main.bib>`
```

Comment:
```suggestion
         :download:`Download BibTeX <../../main.bib>`
```

Reply:
But the `main.bib` and `CITATION.rst` live at the same level of the directory. Is there a better place for the `main.bib` file?

Reply:
But the docs are building the page `docs/project/citation.rst` so the path is relative to that